deployment/modules/gcp/cloudrun/main.tf

```diff
@@ -44,11 +44,12 @@ resource "google_cloud_run_v2_service" "default" {
         "--http_endpoint=:6962",
         "--bucket=${var.bucket}",
         "--spanner_db_path=${local.spanner_log_db_path}",
-        "--spanner_dedup_db_path=${local.spanner_dedup_db_path}",
+        "--spanner_antispam_db_path=${local.spanner_antispam_db_path}",
         "--roots_pem_file=/bin/test_root_ca_cert.pem",
         "--origin=${var.base_name}${var.origin_suffix}",
         "--signer_public_key_secret_name=${var.signer_public_key_secret_name}",
         "--signer_private_key_secret_name=${var.signer_private_key_secret_name}",
+      	"--inmemory_antispam_cache_size=25000000", # About 1GB of memory.
```

Comment:
For potential future CL, wondering if something like https://pkg.go.dev/github.com/dustin/go-humanize might be helpful in making numerical flags understand more friendly representations of large numbers?